### PR TITLE
franklin

### PR DESCRIPTION
**Pull request by blfletcher**

### CODE REVIEW

**Comment by blfletcher** [comment]
Franklin, this is a little hard to follow:
https://github.com/sidorgcb/cashbot-dashboard-react/blob/master/src/components/organisms/SideMenu/index.js#L59

I think a big reason is that the store object has nested objects. You should take a look at how to flatten an object for redux:
https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape

Then you should also pull logic out of the return. Put it above like this:
https://github.com/sidorgcb/cashbot-dashboard-react/blob/shefali/src/components/organisms/OnboardingNavBar/index.js#L49

Sometimes, it may even benefit from being in its own function.

The logic here is similarly difficult to follow:
https://github.com/sidorgcb/cashbot-dashboard-react/blob/master/src/components/molecules/SideMenuGroup/index.js#L21

**Comment by blfletcher** [comment]
You've also got a couple issues with your components
screen shot 2018-07-07 at 8 55 00 pm
screen shot 2018-07-07 at 8 55 08 pm

**Comment by franklinqye** [comment]
Aside from this error:
image
All other errors fixed!
